src/System.Private.Reflection.Core/src/System/Reflection/Runtime/TypeInfos/RuntimeTypeInfo.cs

```diff
@@ -192,9 +193,69 @@ public sealed override MemberInfo[] GetDefaultMembers()
             return defaultMemberName != null ? GetMember(defaultMemberName) : Array.Empty<MemberInfo>();
         }
 
-        public sealed override InterfaceMapping GetInterfaceMap(Type interfaceType)
+        public sealed override InterfaceMapping GetInterfaceMap(Type ifaceType)
```

Comment:
I think most (but maybe not all) of this implementation should move to System.Private.Reflection.Execution. The original architect of the CoreRT reflection stack was trying to keep a separation between "general purpose reflection (Reflection.Core)" and "Redhawk details" (Reflection.Execution). Maybe it's an unnecessary abstraction, but let's try to keep it consistent (e.g. see how `ImplementedIntefaces` below is implemented by calling into `ReflectionCoreExecution.ExecutionEnvironment.TryGetImplementedInterfaces` instead of just calling `RuntimeAugments` directly).

Basically, parts that touch RuntimeAugments should probably be on the Execution side.